src/main/java/com/harrison/messaging/Consumer.java

```diff
@@ -0,0 +1,20 @@
+package com.harrison.messaging;
+
+import org.springframework.amqp.rabbit.annotation.RabbitListener;
+import org.springframework.stereotype.Component;
+
+@Component
+public class Consumer {
+    
+    @RabbitListener(queues = "hello")
+    public void receive(String in) {
+//        String[] arr = in.split(",");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.38.14.86:9000/sonar/coding_rules#rule_key=squid%3ACommentedOutCodeLine)